sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java

```diff
@@ -1563,6 +1565,33 @@ public void testWriteWithBigTableOptionsSetsBulkOptionsAndRetryOptions() {
     assertThat(options.getRetryOptions(), Matchers.equalTo(retryOptionsBuilder.build()));
   }
 
+  @Test
+  public void testWriteWithEnabledDataflowThrottleReporting() {
+    BigtableIO.Write write =
+        defaultWrite.withTableId("TEST-TABLE").withDataflowThrottleReporting(true);
+
+    assertTrue(write.isDataflowThrottleReportingEnabled());
+  }
+
+  @Test
+  public void testWritePipelineWithDataflowThrottleReporting() {
+    final String table = "TEST-TABLE";
+    service.createTable(table);
+
+    BigtableConfig mockConfig =
```

Comment:
I think you can rename this to fakeConfig so it's consistent with your other naming